syncstorage-postgres/src/db/mod.rs

```diff
@@ -161,6 +164,28 @@ impl PgDb {
             })
             .collect()
     }
+
+    async fn erect_tombstone(&mut self, user_id: i64) -> DbResult<()> {
+        let naive_datetime = timestamp_to_naive_datetime(self.timestamp().as_i64())?;
+        diesel::insert_into(user_collections::table)
+            .values((
+                user_collections::user_id.eq(user_id),
+                user_collections::collection_id.eq(TOMBSTONE),
+                user_collections::modified.eq(naive_datetime),
+            ))
+            .on_conflict((user_collections::user_id, user_collections::collection_id))
+            .do_update()
+            .set(user_collections::modified.eq(naive_datetime))
+            .execute(&mut self.conn)
+            .await?;
+        Ok(())
+    }
+}
+
+pub fn timestamp_to_naive_datetime(timestamp_ms: i64) -> DbResult<NaiveDateTime> {
```

Comment:
This function is now in both here and https://github.com/mozilla-services/syncstorage-rs/pull/1903.  I can update whichever PR gets merged second to ensure we don't have a dup.

We could also consider moving the fn to syncstorage-db-common/src/util.rs.  Although the `DbResult` here is a crate level type.  I don't know why syncstorage-mysql and syncstorage-postgres each have their own `DbResult` type though; maybe we can consolidate that as well.